### DISCUSSION TITLE
How do I send my username via openID, not 'authentik default admin'.

### DISCUSSION THREAD

**Discussion by mgrimace** [goauthentik/authentik#7393] [Q&A] [answered]
Hello,

I've set up Vikunja (self-hosted todo app; [link](https://vikunja.io)) following Authentik's integration instructions ([link](https://goauthentik.io/integrations/services/vikunja/)). However, when I log in with Authentik, it sets the username to Default Authentik Admin (and resets it to that on each login, even if I change it in the app). The developper responded that The name is pulled from Authentik. How do I pass my username (or email) using OpenID login to Vikunja?

Thank you

**Comment by ser** [comment, marked as answer]
Please check the configuration details of your App in Authentik, there are options what is passed with authorisation.

**Reply by mgrimace** [reply]
Thanks, turns out I set my display name in Authentik but not my user name (which I didn’t realize were different things until this because I always saw my display name properly).
